cmake/sdlchecks.cmake

```diff
@@ -583,6 +583,26 @@ macro(CheckWayland)
   endif()
 endmacro()
 
+macro(CheckLibDonnell)
+  if(SDL_LIBDONNELL)
+    set(HAVE_LIBDONNELL FALSE)
+	set(LibDonnell_PKG_CONFIG_SPEC donnell)
+	pkg_check_modules(PC_LIBDONNELL IMPORTED_TARGET ${LibDonnell_PKG_CONFIG_SPEC})
+
+    if(PC_LIBDONNELL_FOUND)
+        set(HAVE_LIBDONNELL TRUE)
+		set(HAVE_LIBDONNELL_H TRUE)
+        cmake_push_check_state()
+        list(APPEND CMAKE_REQUIRED_INCLUDES ${PC_LIBDONNELL_INCLUDE_DIRS})
+        list(APPEND CMAKE_REQUIRED_LIBRARIES PkgConfig::PC_LIBDONNELL)
+        cmake_pop_check_state()
```

Comment:
You don't do any build test, so these are not needed.
```suggestion
```

Reply:
Right, thanks for pointing that out, I am not very good at CMake.